node/send_receive/tslint.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
None, there are no defaults and tslint expects a configuration file in your project. Defined a list of rules that I find sensible.